toxav/rtp.c

```diff
@@ -15,25 +15,28 @@
 
 #include "bwcontroller.h"
 
-#include "../toxcore/Messenger.h"
 #include "../toxcore/logger.h"
 #include "../toxcore/mono_time.h"
 #include "../toxcore/util.h"
+#include "../toxcore/net_crypto.h"
+#include "../toxcore/tox_private.h"
+
+Mono_Time *toxav_get_av_mono_time(ToxAV *toxav);
+int rtp_send_custom_lossy_packet(Tox *tox, int32_t friendnumber, const uint8_t *data, uint32_t length);
```

Comment:
`static`?

Reply:
put into toxav_hacks.h and the other one static